src/gobexport/exporter/config/brk.py

```diff
@@ -379,8 +385,8 @@ class AantekeningenExportConfig:
         },
         'BRK_KOT_ID': 'rustOpKadastraalobject.[0].identificatie',
         'KOT_KADASTRALEGEMCODE_CODE':
-            'rustOpKadastraalobject.[0].aangeduidDoorKadastralegemeentecode.broninfo.omschrijving',
-        'KOT_SECTIE': 'aangeduidDoorKadastralesectie.[0].bronwaarde',
+            'rustOpKadastraalobject.[0].aangeduidDoorKadastralegemeentecode.[0].broninfo.broninfo.omschrijving',
+        'KOT_SECTIE': 'rustOpKadastraalobject.[0].aangeduidDoorKadastralesectie.[0].code',
```

Comment:
I don't think this will work. References to relations are always moved to the root, so nested relations are always referenced as the inner relation: rustOpkadastraalobject.[0].aangeduidDoorKadastralesectie.[0].code will simply become aangeduidDoorKadastralesectie.[0].code